DevEdu.API/DevEdu.API/Controllers/TaskController.cs

```diff
@@ -138,75 +140,104 @@ public void DeleteTagFromTask(int taskId, int tagId)
 
         // api/task/{taskId}/student/{studentId}
         [HttpPost("{taskId}/student/{studentId}")]
-        public void AddStudentAnswerOnTask(int taskId, int studentId, [FromBody] StudentAnswerOnTaskInputModel inputModel)
+        [Description("Add student answer on task")]
+        [ProducesResponseType(StatusCodes.Status201Created)]

```

Comment:
we must return the added entity here
so please add the correct typeof here
and change the return type of this method to StudentAnswerOnTaskOutputModel